rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -578,7 +578,7 @@ void main_window::InstallPup(const QString& dropPath)
 		version_string.erase(version_pos);
 	}
 
-	const std::string cur_version = "4.84";
+	const std::string cur_version = "4.85.1";
```

Comment:
```suggestion
	const std::string cur_version = "4.85";
```
Shouldn't this be 4.85? I don't seem to find this .1 anywhere. And the update page specifically says 4.85